src/pages/Photos.jsx

```diff
@@ -0,0 +1,170 @@
+import React, { Fragment } from 'react';
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+import { Card } from 'reactstrap';
+
+import avatar1 from '../assets/images/avatars/avatar1.jpg';
+import avatar2 from '../assets/images/avatars/avatar2.jpg';
+import avatar3 from '../assets/images/avatars/avatar3.jpg';
+import avatar4 from '../assets/images/avatars/avatar4.jpg';
+import avatar5 from '../assets/images/avatars/avatar5.jpg';
+import avatar6 from '../assets/images/avatars/avatar6.jpg';
+import avatar7 from '../assets/images/avatars/avatar7.jpg';
+
+const Photo = () =>
+  <Fragment>
+    <Card className="card-box mb-5 p-3 text-center">
+      <div className="my-3">
+        <h6 className="font-weight-bold font-size-lg mb-1 text-black">
+          Photo Gallery
+        </h6>
+        <p className="text-black-50 mb-0">
+          Photo display gallery.
+        </p>
+      </div>
+      <div className="d-flex flex-row flex-wrap justify-content-center">
+        <div className="position-relative px-5 py-3">
+          <div className="divider-v divider-v-lg" />
+          <div className="avatar-icon-wrapper rounded-circle d-80 mx-auto">
+            <div className="d-block p-0 avatar-icon-wrapper rounded-circle m-0">
+              <div className="rounded-circle overflow-hidden">
+                <img alt="..." className="img-fluid" src={avatar1} />
```

Comment:
`...` is not sufficient alt text for an image. Normally the alt text would be the name of the person in the photo such as `alt="Headshot of John Smith"`